frontend/src/layout/header/Logo.tsx

```diff
@@ -42,7 +43,7 @@ export const Logo: React.FC = () => {
                 borderRadius: 4,
                 outlineOffset: -2,
                 ":hover": { outline: "2px solid var(--grey80)" },
-                ":focus": { outline: "2px solid var(--accent-color)" },
+                ...focusStyle({ inset: true }),
```

Comment:
So the `outlineOffset` above can be removed, right?

Reply:
Yeah, well in this case it is actually necessary for the hover outline, but it can definitely be moved inside the `:hover` state (or that state could as above be removed completely)

Reply:
Oh right, I didn't consider that the outline is also used on hover